docs/en/latest/plugins/aws-lambda.md

```diff
@@ -136,6 +137,43 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 
 **Note**: This approach assumes you already have an iam user with the programmatic access enabled and required permissions (`AmazonAPIGatewayInvokeFullAccess`) to access the endpoint.
 
+### Plugin with Path Forwarding
+
+AWS Lambda plugin supports url path forwarding while proxying request to the modified upstream (AWS Gateway URI endpoint). With that being said, any extension to the path of the base request APISIX gateway URI gets "appended" (path join) to the `function_uri` specified in the plugin configuration. To test this behavior, see the below example:
```

Comment:
Better to emphasize that the uri of the route needs to be ended with `*`.

Reply:
Done